connectors/migrations/db/migration_109.sql

```diff
@@ -0,0 +1,7 @@
+-- Migration created on Dec 04, 2025
+UPDATE "public"."slack_configurations"
+SET "botEnabled" = false
+WHERE "botEnabled" = true
+AND "connectorId" IN (
+  SELECT id FROM "connectors" WHERE "type" = 'slack'
+);
```

Comment:
Do these already have a "slack_bot" connector or is it going to break the bot for them ?

Reply:
I think they may or may not have one, but either way, this flag is not what makes it work. The slack_bot connector has its own botEnabled flag that matters.

Reply:
Today the bot is not working for them, best case they receive a deprecated message if they try to ping the dust app from slack inviting them so use slack bot instead